src/js/angular/sparql/directives/query-editor.directive.js

```diff
@@ -272,7 +272,7 @@ function queryEditorDirective($timeout, localStorageService, $location, toastr,
                 const doExecute = function () {
                     scope.currentTabConfig.queryType = "UPDATE";
 
-                    SparqlService.getRepositorySize()
+                    RDF4JRepositoriesRestService.getRepositorySize()
```

Comment:
Maybe also extract line 288 $http.post('rest/connectors/check')

Reply:
I actually didn't extracted 100% of the http calls in these rest clients. Some of those which I left were the connectors and locations and also some of the similarity endpoints as you noticed in the other comment. I could extract them in another smaller MR or here.